packages/frenetic/frenetic.5.0.5/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "The Frenetic Programming Language and Runtime System"
+maintainer: "Steffen Smolka <smolka@cs.cornell.edu>"
+authors: "Arjun Guha <arjun@cs.umass.edu>, Nate Foster <jnfoster@cs.cornell.edu>, Steffen Smolka <smolka@cs.cornell.edu>"
+homepage: "http://frenetic-lang.org"
+bug-reports: "https://github.com/frenetic-lang/frenetic/issues"
+dev-repo: "git+https://github.com/frenetic-lang/frenetic.git"
+build: [
```

Comment:
Can you please add the license in spdx form: https://spdx.org/licenses/

Reply:
Thanks. Fixed locally.